src/routes/products/storage/(components)/OpenSource.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+    import { classNames } from '$lib/utils/classnames';
+
+    const items = [
+        {
+            title: 'Join the community and share knowledge',
+            content: 'Lorem ipsum dolor sit amet.',
+            icon: '/images/icons/gradients/community.svg'
+        },
+        {
+            title: 'Self-host or go Cloud',
+            content: 'Lorem ipsum dolor sit amet consectetur. Nullam ligula blandit.',
+            icon: '/images/icons/gradients/self-hosted.svg'
+        },
+        {
+            title: 'No vendor lock-in',
+            content: 'Lorem ipsum dolor sit amet consectetur. Nullam ligula blandit.',
+            icon: '/images/icons/gradients/lock.svg'
+        }
+    ];
+</script>
+
+<div class="web-big-padding-section-level-2 light">
+    <div class="container">
+        <div class="web-hero is-center web-u-max-width-580 mx-auto">
+            <h2 class="web-display web-u-color-text-primary">Powered by open source</h2>
```

Comment:
I see we type here "open source", on the site I also see "open-source" and "Open Source", do we want to use the same way everywhere? cc @LauraDuRy

Reply:
Good question!

Reply:
Going to get this shipped for now, can edit later.